Tasks/JenkinsQueueJobV2/job.ts

```diff
@@ -131,6 +131,14 @@ export class Job {
         this.working = false;
     }
 
+    
+    private RetryConnection(): void {
+        const thisJob: Job = this;
+        thisJob.queue.TaskOptions.retryCount--;
+        thisJob.consoleLog(`Connection error. Retrying again in ${thisJob.queue.TaskOptions.delayBetweenRetries} seconds`);
```

Comment:
Could it make sense to write remaining attempts?

Reply:
With changing retry number, it would seem better to show it, yeah.